CanalTP/Chaos
# [Feature] multi messages on same channel

to test,

Try to C R U a disruption with impact(s) with multiple messages of the same type with different contents. Every actions should be OK, no error.



relates to BO-312
